lib/resources/r4/appointment.yaml

```diff
@@ -25,6 +25,23 @@ fields:
     }
   note: When creating a new appointment, the status must be `booked` or `proposed`.
 
+- name: serviceCategory
+  required: 'No'
+  type: CodeableConcept
+  description: A broad categorization of the service that is to be performed during this appointment.
+  binding:
+    description:  Additional details about the categorization of the service.
```

Comment:
Not sure on the description here. Any thought?

Reply:
This description sounds good to me